src/components/moto/moto_list.css

```diff
@@ -1,4 +1,4 @@
-ul {
+/* ul {
   list-style-type: none;
   padding: 0;
 }
```

Comment:
If you are not going to use this style anymore, please drop this and all similar files